From 60a5f953f747e1e9e05a40157b651cba8ea57b91 Mon Sep 17 00:00:00 2001
From: Dengke Du <dengke.du@windriver.com>
Date: Mon, 11 Sep 2017 02:21:55 -0400
Subject: [PATCH 3/5] ovmf: enable long path file

Upstream-Status: Pending
Signed-off-by: Dengke Du <dengke.du@windriver.com>

diff --git a/BaseTools/Source/C/Common/CommonLib.h b/BaseTools/Source/C/Common/CommonLib.h
index 0f05d88db2..ccda329c82 100644
--- a/BaseTools/Source/C/Common/CommonLib.h
+++ b/BaseTools/Source/C/Common/CommonLib.h
@@ -21,7 +21,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #ifdef PATH_MAX
 #define MAX_LONG_FILE_PATH PATH_MAX
 #else
-#define MAX_LONG_FILE_PATH 500
+#define MAX_LONG_FILE_PATH 1023
 #endif
 
 #define MAX_UINT64 ((UINT64)0xFFFFFFFFFFFFFFFFULL)
