diff -ur git.orig/Cargo.lock git/Cargo.lock
--- git.orig/Cargo.lock	2020-11-05 14:05:07.085461998 +0000
+++ git/Cargo.lock	2020-11-05 14:11:13.658221475 +0000
@@ -232,6 +232,7 @@
  "thiserror",
  "vhost_user_block",
  "vhost_user_net",
+ "vm-memory",
  "vmm",
  "vmm-sys-util",
  "wait-timeout",
@@ -1593,7 +1594,8 @@
 [[package]]
 name = "vm-memory"
 version = "0.3.0"
-source = "git+https://github.com/cloud-hypervisor/vm-memory?branch=ch#6f2f7107f29a8f2b66b64a849aeace20783ec8f3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "013d6d0701ad7c68a58e752a8a9736dff957073af5a567c2cbbe0e382657059d"
 dependencies = [
  "arc-swap",
  "libc",
diff -ur git.orig/Cargo.toml git/Cargo.toml
--- git.orig/Cargo.toml	2020-11-05 14:05:07.085461998 +0000
+++ git/Cargo.toml	2020-11-05 14:18:08.963514271 +0000
@@ -29,20 +29,11 @@
 vmm = { path = "vmm" }
 vmm-sys-util = "0.6.1"
 wait-timeout = "0.2.0"
+vm-memory = "0.3.0"
 
 [build-dependencies]
 clap = { version = "2.33.3", features = ["wrap_help"] }
 
-# List of patched crates
-[patch.crates-io]
-vm-memory = { git = "https://github.com/cloud-hypervisor/vm-memory", branch = "ch" }
-[patch.'https://github.com/rust-vmm/vhost']
-vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
-[patch.'https://github.com/rust-vmm/kvm-ioctls']
-kvm-ioctls = { git = "https://github.com/cloud-hypervisor/kvm-ioctls", branch = "ch" }
-[patch.'https://github.com/rust-vmm/kvm-bindings']
-kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch", features = ["with-serde", "fam-wrappers"] }
-
 [dev-dependencies]
 ssh2 = "0.8.2"
 dirs = "3.0.1"
diff --git a/fuzz/Cargo.toml b/fuzz/Cargo.toml
index 2089ce2f..48c6b487 100644
--- a/fuzz/Cargo.toml
+++ b/fuzz/Cargo.toml
@@ -10,7 +10,7 @@ edition = "2018"
 cargo-fuzz = true
 
 [dependencies]
-kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "master" }
+kvm-ioctls = { git = "https://github.com/cloud-hypervisor/kvm-ioctls", branch = "ch" }
 libc = "0.2.72"
 libfuzzer-sys = "0.3"
 qcow = { path = "../qcow" }
diff --git a/hypervisor/Cargo.toml b/hypervisor/Cargo.toml
index 8bfd695a..06032667 100644
--- a/hypervisor/Cargo.toml
+++ b/hypervisor/Cargo.toml
@@ -14,8 +14,8 @@ arc-swap = ">=0.4.4"
 thiserror = "1.0"
 libc = "0.2.80"
 log = "0.4.11"
-kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "master" }
-kvm-bindings = { git = "https://github.com/rust-vmm/kvm-bindings", branch = "master", features = ["with-serde", "fam-wrappers"] }
+kvm-ioctls = { git = "https://github.com/cloud-hypervisor/kvm-ioctls", branch = "ch" }
+kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch", features = ["with-serde", "fam-wrappers"] }
 serde = {version = ">=1.0.27", features = ["rc"] }
 serde_derive = ">=1.0.27"
 serde_json = ">=1.0.9"
diff --git a/vhost_user_backend/Cargo.toml b/vhost_user_backend/Cargo.toml
index 41833d84..eaefeb75 100644
--- a/vhost_user_backend/Cargo.toml
+++ b/vhost_user_backend/Cargo.toml
@@ -15,4 +15,4 @@ virtio-bindings = "0.1.0"
 vm-memory = "0.3.0"
 vm-virtio = { path = "../vm-virtio" }
 vmm-sys-util = ">=0.3.1"
-vhost_rs = { git = "https://github.com/rust-vmm/vhost", branch = "master", package = "vhost", features = ["vhost-user-slave"] }
+vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
diff --git a/vhost_user_block/Cargo.toml b/vhost_user_block/Cargo.toml
index 71e35b0f..21e04faa 100644
--- a/vhost_user_block/Cargo.toml
+++ b/vhost_user_block/Cargo.toml
@@ -13,7 +13,7 @@ log = "0.4.11"
 option_parser = { path = "../option_parser" }
 qcow = { path = "../qcow" }
 vhost_user_backend = { path = "../vhost_user_backend" }
-vhost_rs = { git = "https://github.com/rust-vmm/vhost", branch = "master", package = "vhost", features = ["vhost-user-slave"] }
+vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
 virtio-bindings = "0.1.0"
 vm-memory = "0.3.0"
 vmm-sys-util = ">=0.3.1"
diff --git a/vhost_user_fs/Cargo.toml b/vhost_user_fs/Cargo.toml
index ca478609..1c32c122 100644
--- a/vhost_user_fs/Cargo.toml
+++ b/vhost_user_fs/Cargo.toml
@@ -16,7 +16,7 @@ seccomp = { git = "https://github.com/firecracker-microvm/firecracker", tag = "v
 tempdir = "0.3.7"
 virtio-bindings = { version = "0.1", features = ["virtio-v5_0_0"]}
 virtio-devices = { path = "../virtio-devices" }
-vhost_rs = { git = "https://github.com/rust-vmm/vhost", branch = "master", package = "vhost", features = ["vhost-user-slave"] }
+vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
 vhost_user_backend = { path = "../vhost_user_backend"}
 vm-memory = "0.3.0"
 vm-virtio = { path = "../vm-virtio" }
diff --git a/virtio-devices/Cargo.toml b/virtio-devices/Cargo.toml
index 9514861d..3ea244a3 100644
--- a/virtio-devices/Cargo.toml
+++ b/virtio-devices/Cargo.toml
@@ -26,7 +26,7 @@ serde_derive = ">=1.0.27"
 serde_json = ">=1.0.9"
 tempfile = "3.1.0"
 vfio-ioctls = { git = "https://github.com/cloud-hypervisor/vfio-ioctls", branch = "ch" }
-vhost_rs = { git = "https://github.com/rust-vmm/vhost", branch = "master", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
+vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
 virtio-bindings = { version = "0.1", features = ["virtio-v5_0_0"]}
 vm-allocator = { path = "../vm-allocator" }
 vm-device = { path = "../vm-device" }
diff --git a/vhost_user_net/Cargo.toml b/vhost_user_net/Cargo.toml
index 7b26298c..c0335fce 100644
--- a/vhost_user_net/Cargo.toml
+++ b/vhost_user_net/Cargo.toml
@@ -12,7 +12,7 @@ log = "0.4.11"
 net_util = { path = "../net_util" }
 option_parser = { path = "../option_parser" }
 vhost_user_backend = { path = "../vhost_user_backend" }
-vhost_rs = { git = "https://github.com/rust-vmm/vhost", branch = "master", package = "vhost", features = ["vhost-user-slave"] }
+vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "ch", package = "vhost", features = ["vhost-user-master", "vhost-user-slave"] }
 virtio-bindings = "0.1.0"
 vm-memory = "0.3.0"
 vmm-sys-util = ">=0.3.1"
